src/ucp/core/ucp_ep.inl

```diff
@@ -289,4 +289,17 @@ static UCS_F_ALWAYS_INLINE int ucp_ep_use_indirect_id(ucp_ep_h ep)
     UCS_STATIC_ASSERT(sizeof(ep->flags) <= sizeof(int));
     return ep->flags & UCP_EP_FLAG_INDIRECT_ID;
 }
+
+static UCS_F_ALWAYS_INLINE void ucp_ep_cnt_inc(uint8_t *cnt)
```

Comment:
1. convert this function to macro 
2. rename to ucp_ep_assert_counter_inc(_counter)